custom_components/mawaqit/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "mawaqit_prayer_times",
```

Comment:
We can rename the domain and name  MAWAQIT  only ?

Reply:
done in v2.2. current users of the integrations need to completely remove the old version